README.md

```diff
@@ -22,11 +22,27 @@ Run Test Suite in Debug Mode:
 
 Database
 ========
-
 ```
 psql --command="create database forum;" && psql --echo-all --dbname=forum --file=tables.sql
 ```
 
+Connect to the database and add one column to the `forum_user` table:
+```
+alter table forum_user add column is_v2 boolean;
+```
+
 Hot Reload
 ==========
 `npm run start` does 'hot reloading' by default. This is subject to change.
+
```

Comment:
all of these additions should come before the mention of hot reload (the hot reload mention is an afterthought, not a super important piece of info like the rest of this is)